platform/ios/INSTALL.md

```diff
@@ -111,6 +111,8 @@ For instructions on installing stable release versions of the Mapbox Maps SDK fo
 To test pre-releases of the dynamic framework, directly specify the version in your `Podfile`:
 
 ```rb
+use frameworks!
```

Comment:
What do you think if we specify the use of this flag as a step and mention that is required? Instead of adding it individually for each cocoapods call.